data/json/items/comestibles/med.json

```
@@ -2042,7 +2042,7 @@
     "type": "COMESTIBLE",
     "comestible_type": "MED",
     "name": { "str": "cattail jelly", "str_pl": "cattail jellies" },
-    "description": "Clear jelly harvested from cattail, has antiseptic properties and works as a painkiller.",
+    "description": "Clear jelly harvested from cattail. Cancbe used to protect a wound from contamination and works as a painkiller.",
```

```suggestion
    "description": "Clear jelly harvested from cattail.  Can be used to protect a wound from contamination and works as a painkiller.",
```

The bots *also* insist on two spaces after a period. I get this wrong every single time.

Otherwise this looks like a sensible change.